packages/ag-charts-community/src/chart/toolbar/toolbarStyles.css

```diff
@@ -130,6 +130,7 @@
     pointer-events: all;
     position: relative;
     white-space: nowrap;
+    --icon-color: var(--ag-charts-toolbar-foreground-color);
```

Comment:
I suggest `--stroke-width-color` or similar, since icons are something else and don't use this color var.